app/src/NativeBase/Assets/Icons/selection.json

```diff
@@ -0,0 +1,126 @@
+{
```

Comment:
Might be good to name this file something like `icomoon_selection.json` so it's even clearer what it's for (or create a separate sub-directory for the Icomoon files so they can sit together)?